spec/runners/cmap/verifier.rb

```diff
@@ -41,7 +41,9 @@ def verify_hashes(actual, expected)
           end
         end
 
-        expect(actual_modified).to eq(expected)
+        expected.each do |k, v|
```

Comment:
The spec describes the match pseudocode [here](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/tests/README.rst#L158):
```
 For every key/value in expected
        Assert that actual[key] MATCHES value
```

Reply:
If it's in the spec...